foodhub_product_fields/tests/test_product.py

```diff
@@ -27,13 +26,7 @@ def setUpClass(cls):
                 "volume": 4,
                 "weight": 2,
                 "seller_ids": [
-                    (
-                        0,
-                        False,
-                        {
-                            "name": cls.supplier.id,
-                        },
-                    )
+                    (0, 0, {"partner_id": cls.supplier.id, "product_code": "ASUCODE"}),
```

Comment:
`product_code` is not required, why did you add it ?

Reply:
just a copy paste error, removed